app/pages/lab/mobile/mobile_previewer.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { zooTheme } from '../../../theme'
+
+export default class MobilePreviewer extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
`super(props)` -> the "props" argument isn't necessary unless props are being manipulated in the constructor.

Reply:
I could probably just get rid of the constructor. I don't use it

Reply:
🖕